src/theme.js

```diff
@@ -220,6 +220,13 @@ function getTheme({ theme, name }) {
       'terminal.ansiBrightCyan': color.ansi.cyanBright,
       'terminal.ansiBrightWhite': color.ansi.whiteBright,
 
+      "editorBracketHighlight.foreground1": lightDark(scale.blue[5], scale.blue[2]),
+      "editorBracketHighlight.foreground2": lightDark(scale.orange[5], scale.orange[2]),
+      "editorBracketHighlight.foreground3": lightDark(scale.purple[5], scale.purple[2]),
+      "editorBracketHighlight.foreground4": lightDark(scale.blue[5], scale.blue[2]),
+      "editorBracketHighlight.foreground5": lightDark(scale.orange[5], scale.orange[2]),
+      "editorBracketHighlight.foreground6": lightDark(scale.purple[5], scale.purple[2]),
```

Comment:
Thanks for the PR. I guess the default in VS Code and the GitHub classic themes only use 3 colors that get repeated. But I wonder if we could use 6 unique colors before the repetition starts? Maybe something like:

```suggestion      
      "editorBracketHighlight.foreground1": lightDark(scale.blue[5], scale.blue[2]),
      "editorBracketHighlight.foreground2": lightDark(scale.green[5], scale.green[2]),
      "editorBracketHighlight.foreground3": lightDark(scale.yellow[5], scale.yellow[2]),
      "editorBracketHighlight.foreground4": lightDark(scale.red[5], scale.red[2]),
      "editorBracketHighlight.foreground5": lightDark(scale.pink[5], scale.pink[2]),
      "editorBracketHighlight.foreground6": lightDark(scale.purple[5], scale.purple[2]),
      "editorBracketHighlight.unexpectedBracket.foreground": color.fg.muted, // gray
```

VS Code default | Suggested change
--- | ---
![Screen Shot 2022-07-13 at 12 19 07](https://user-images.githubusercontent.com/378023/178643954-968884d8-ae3c-4da6-8360-842eee9d0048.png) | ![Screen Shot 2022-07-13 at 12 19 56](https://user-images.githubusercontent.com/378023/178643960-95ac355d-5f7c-45b4-9e39-5da38c7d2ff6.png)

Reply:
Ok, let's get this into the next release. 🚢